src/paperbot/api/routes/research.py

```
@@ -645,17 +672,65 @@ def add_paper_feedback(req: PaperFeedbackRequest):
+                abstract=req.paper_abstract or "",
+                authors=req.paper_authors or [],
+                semantic_scholar_id=req.paper_id if source == HarvestSource.SEMANTIC_SCHOLAR else None,
+                arxiv_id=req.paper_id if source == HarvestSource.ARXIV else None,
+                openalex_id=req.paper_id if source == HarvestSource.OPENALEX else None,
+                year=req.paper_year,
+                venue=req.paper_venue,
+                citation_count=req.paper_citation_count or 0,
+                url=req.paper_url,
+            )
+            Logger.info("Calling paper store to upsert paper", file=LogFiles.HARVEST)
+            new_count, _ = paper_store.upsert_papers_batch([paper])
```

Ruff (0.15.0)

[warning] Unpacked variable `new_count` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)